docs/en/connector-v2/sink/Doris.md

```diff
@@ -17,67 +30,130 @@ Version Supported
+| TINYINT         | TINYINT                                 |
+| SMALLINT        | SMALLINT<br/>TINYINT                    |
+| INT             | INT<br/>SMALLINT<br/>TINYINT            |
+| BIGINT          | BIGINT<br/>INT<br/>SMALLINT<br/>TINYINT |
+| LARGEINT        | BIGINT<br/>INT<br/>SMALLINT<br/>TINYINT |
+| FLOAT           | FLOAT                                   |
+| DOUBLE          | DOUBLE<br/>FLOAT                        |
+| DECIMAL         | DECIMAL<br/>DOUBLE<br/>FLOAT            |
+| DATE            | DATE                                    |
+| DATETIME        | TIMESTAMP                               |
+| CHAR            | STRING                                  |
+| VARCHAR         | STRING                                  |
+| STRING          | STRING                                  |
+| ARRAY           | ARRAY                                   |
+| MAP             | MAP                                     |
+| JSON            | STRING                                  |
+| HLL             | Not supported yet                       |
+| BITMAP          | Not supported yet                       |
+| QUANTILE_STATE  | Not supported yet                       |
+| STRUCT          | Not supported yet                       |
 
-### password [string]
 
-`Doris` user password
-
-### table.identifier [string]
-
-The name of `Doris` table
-
-### sink.label-prefix [string]
-
-The label prefix used by stream load imports. In the 2pc scenario, global uniqueness is required to ensure the EOS semantics of SeaTunnel.
-
-### sink.enable-2pc [bool]
-
-Whether to enable two-phase commit (2pc), the default is true, to ensure Exactly-Once semantics. For two-phase commit, please refer to [here](https://doris.apache.org/docs/dev/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD).
-
-### sink.enable-delete [bool]
+#### Supported import data formats
 
-Whether to enable deletion. This option requires Doris table to enable batch delete function (0.15+ version is enabled by default), and only supports Unique model. you can get more detail at this link:
+The supported formats include CSV and JSON
 
-https://doris.apache.org/docs/dev/data-operate/update-delete/batch-delete-manual
+## Task Example
 
-### doris.config [map]
+### Simple:
 
-The parameter of the stream load `data_desc`, you can get more detail at this link:
+> This example defines a SeaTunnel synchronization task that automatically generates data through FakeSource and sends it to Doris Sink,FakeSource simulates CDC data with schema pk_id (bigint type), name (string type), and score (int type),Doris needs to create a table sink named test.e2e_table_sink and a corresponding table for it.
 
-https://doris.apache.org/docs/dev/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD
+```hocon
+env {
+  parallelism = 1
+  job.mode = "BATCH"
+  checkpoint.interval = 10000
+}
 
-#### Supported import data formats
+source {
```

Comment:
```
FakeSource {
  row.num = 10
  map.size = 10
  array.size = 10
  bytes.length = 10
  string.length = 10
  schema = {
    fields {
      c_map = "map<string, array<int>>"
      c_array = "array<int>"
      c_string = string
      c_boolean = boolean
      c_tinyint = tinyint
      c_smallint = smallint
      c_int = int
      c_bigint = bigint
      c_float = float
      c_double = double
      c_decimal = "decimal(30, 8)"
      c_null = "null"
      c_bytes = bytes
      c_date = date
      c_timestamp = timestamp
      c_row = {
        c_map = "map<string, map<string, string>>"
        c_array = "array<int>"
        c_string = string
        c_boolean = boolean
        c_tinyint = tinyint
        c_smallint = smallint
        c_int = int
        c_bigint = bigint
        c_float = float
        c_double = double
        c_decimal = "decimal(30, 8)"
        c_null = "null"
        c_bytes = bytes
        c_date = date
        c_timestamp = timestamp
      }
    }
  }
  result_table_name = "fake"
}

```
This data source type is more comprehensive

Reply:
Not every version of doris supports complex types

Reply:
PTAL: @TaoZex @EricJoy2048